frontend/src/Game/index.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { View } from 'react-native';
 import styles from '../common/styles';
 import Menu from '../Menu';
-import Buildings from '../Buildings';
+import Leaderboard from '../Leaderboard/index';
```

Comment:
```suggestion
import Leaderboard from '../Leaderboard';
```
actually we don't need this index here.